src/coreclr/nativeaot/Runtime/EHHelpers.cpp

```diff
@@ -195,7 +211,7 @@ FCIMPL3(void, RhpCopyContextFromExInfo, void * pOSContext, int32_t cbOSContext,
 }
 FCIMPLEND
 
-#if defined(HOST_AMD64) || defined(HOST_ARM) || defined(HOST_X86) || defined(HOST_ARM64)
+#if defined(HOST_AMD64) || defined(HOST_ARM) || defined(HOST_X86) || defined(HOST_ARM64) || defined(HOST_LOONGARCH64)
```

Comment:
```suggestion
```
Can we just delete this ifdef? The code seems to be reasonably portable.

Reply:
Ok, I've deleted these two ifdef. 
But I'm not sure if my change was right, expecting your reviewing.
Thanks!